packages/firebase-functions/README.md

```diff
@@ -48,6 +48,45 @@ firebase()
 	});
 ```
 
+## Regional Cloud Functions
+Cloud Functions are _regional_, which means the infrastructure that runs your Cloud Function is located in specific regions.
+
+<<<<<<< HEAD
+By default, functions run in the _us-central1_ region. View the [supported regions](https://firebase.google.com/docs/functions/locations).
+=======
+By default, functions run in the _us-central1_ region. View the [supported regions](https://firebase.google.com/docs/functions/locations)
+>>>>>>> e0602081644464c60610caa66d227ba3dfcf2c57
```

Comment:
This looks great @Langers8 thank you - possible to cleanup this unmerged conflict here?

Reply:
Good spot - that looked messy.  I've cleaned it up. Thanks!